### PR TITLE
[WIP] add placeholder docs for disabling node ports for Service Type=LB

### PR DESCRIPTION
**Pull request by andrewsykim**
Signed-off-by: Andrew Sy Kim <kim.andrewsy@gmail.com>

Placeholder docs for KEP https://github.com/kubernetes/enhancements/issues/1864 

Documents that node ports can now be optionally disabled for Service Type=LoadBalancer.

cc @uablrek

### CODE REVIEW

**Comment by andrewsykim** [comment]
PR for this: https://github.com/kubernetes/kubernetes/pull/92744

**Comment by annajung** [comment]
Hey @andrewsykim please change the base to `dev-1.20` branch

**Comment by kcmartin** [comment]
/hold cancel
PR has merged https://github.com/kubernetes/kubernetes/pull/92744

**Comment by kcmartin** [comment]
@andrewsykim 
Hi, quick reminder to change the base to `dev-1.20` branch

**Comment by kcmartin** [comment]
Hi @andrewsykim 

A friendly reminder that Docs "Ready for Review" Deadline is this Monday, Nov. 23rd for the 1.20 release.

Please reach out if you have any questions.

**Comment by annajung** [comment]
Exception requested/granted. PR will be ready for review by EOD Wednesday the 25th

File: content/en/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -604,6 +604,10 @@ Specify the assigned IP address as loadBalancerIP. Ensure that you have updated
 
 {{< /note >}}
 
+#### Disabling Node Ports
+
+TODO
+
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/service.md:610]: We should also document that a LoadBalancer service comes, by default, with a NodePort.

**Review by sftim — COMMENTED** [review]
Do we also want to revise https://kubernetes.io/docs/tutorials/services/source-ip/?

File: content/en/docs/concepts/services-networking/service.md (hunk 1 above)
Comment (andrewsykim) [reply content/en/docs/concepts/services-networking/service.md:610]: Good point, will add some words for that

**Comment by andrewsykim** [comment]
> Do we also want to revise https://kubernetes.io/docs/tutorials/services/source-ip/?

I don't think so since we haven't changed any defaults around node ports so everything there still seems accurate.

**Comment by andrewsykim** [comment]
Closing in favor of https://github.com/kubernetes/website/pull/25228

**Pull request closed without merging**